app/src/main/java/org/wikipedia/util/JobsUtil.kt

```diff
@@ -0,0 +1,16 @@
+package org.wikipedia.util
+
+import android.app.job.JobInfo
+import android.app.job.JobScheduler
+import android.content.ComponentName
+import android.content.Context
+import org.wikipedia.notifications.PollNotificationService
+
+object JobsUtil {
+    fun schedulePollNotificationJob(context: Context) {
```

Comment:
Can this be placed in a companion object of `PollNotificationService` itself, instead of a separate file?